### PR TITLE
OCM-2437 | feat: Added support for /api/clusters_mgmt/v1/aws_inquiries/sts_account_roles to the SDK

### PR DESCRIPTION
**Pull request by robpblake**
This PR adds support for `/api/clusters_mgmt/v1/aws_inquiries/sts_account_roles` to the SDK. 

The backend is already implemented in CS, this exposes the functionality via the SDK.

### CODE REVIEW

**Comment by robpblake** [comment]
This is picking up the additional changes from #842, so ideally we need that merged first and then I can rebase.

File: clustersmgmt/v1/environment_type_json.go [hunk 1]
```diff
@@ -93,26 +102,30 @@ func readEnvironment(iterator *jsoniter.Iterator) *Environment {
 			break
 		}
 		switch field {
+		case "backplane_url":
```
Comment (robpblake) [inline clustersmgmt/v1/environment_type_json.go:105]: Changes from #842, which should be merged first.

**Comment by oriAdler** [comment]
@robpblake we need to update the `CHANGES.md` file as well.
Similar PR https://github.com/openshift-online/ocm-sdk-go/pull/786/files#diff-d975bf659606195d2165918f93e1cf680ef68ea3c9cab994f033705fea8238b2

**Comment by oriAdler** [comment]
We also need to update the `version.go` file.

File: clustersmgmt/v1/environment_type_json.go (hunk 1 above)
Comment (robpblake) [reply clustersmgmt/v1/environment_type_json.go]: These changes have been removed from this PR as they are no longer required.

**Comment by robpblake** [comment]
@oriAdler Updated `version.go` and `CHANGES.md` as requested. Thanks for the pointer 👍

**Review by oriAdler — APPROVED** [review]

**Pull request merged**